packages/p2p-v2/src/components/BuySellForm/BuySellForm.tsx

```diff
@@ -0,0 +1,167 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import Modal from 'react-modal';
+import { BUY_SELL, RATE_TYPE } from '@/constants';
+import { removeTrailingZeros, roundOffDecimal, setDecimalPlaces } from '@/utils';
+import { p2p } from '@deriv/api';
+import { Divider, InlineMessage, Text, useDevice } from '@deriv-com/ui';
+import { BuySellAmount } from '../BuySellAmount';
+import { BuySellData } from '../BuySellData';
+import { BuySellFormFooter } from '../BuySellFormFooter';
+import { BuySellFormHeader } from '../BuySellFormHeader';
+import { customStyles } from '../Modals/helpers';
+import './BuySellForm.scss';
+
+type NonUndefinedValues<T> = {
+    [K in keyof T]-?: Exclude<T[K], undefined>;
+};
```

Comment:
@nada-deriv, this looks like something that belongs in `utils` or something like that 🤔  
```suggestion
type NonUndefinedValues<T> = {
    [K in keyof T]-?: Exclude<T[K], undefined>;
};
```

Reply:
didn't add it there since it was not being reused..